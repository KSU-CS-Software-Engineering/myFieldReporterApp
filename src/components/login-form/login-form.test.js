// login-form.test.js
import React from 'react';
import LoginForm from './login-form';
import renderer from 'react-test-renderer';
import * as firebase from 'firebase';
import FirebaseServer from 'firebase-server';

 
  var config = {
      apiKey: "AIzaSyAHaFwzXQfOCzLRc61T7buHYtfMucIkwF8",
      authDomain: "myfields-researcher.firebaseapp.com",
      databaseURL: "test.firebase.localhost:5000",
      projectId: "myfields-researcher",
      storageBucket: "myfields-researcher.appspot.com",
      messagingSenderId: "545497609582"
  };
  firebase.initializeApp(config);

test('Renders a login form', () => {
  const component = renderer.create(
    <LoginForm />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});