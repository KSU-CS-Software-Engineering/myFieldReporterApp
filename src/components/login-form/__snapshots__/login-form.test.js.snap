// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a login form 1`] = `
<div
  className="login-form-container"
>
  <h1>
    Login
  </h1>
  <div
    className="message"
  />
  <div
    id="login-container"
  >
    <input
      className="first-line"
      name="email"
      onChange={[Function]}
      placeholder="Email Address"
      required={true}
      type="email"
      value=""
    />
    <input
      className="second-line"
      name="password"
      onChange={[Function]}
      onKeyPress={[Function]}
      placeholder="Password"
      required={true}
      type="password"
      value=""
    />
    <div
      className="email-icon first-icon"
    />
    <div
      className="password-icon second-icon"
    />
  </div>
  <button
    onClick={[Function]}
  >
    Login
  </button>
  <button
    onClick={[Function]}
  >
    New User
  </button>
  <a
    onClick={[Function]}
  >
    Forgot Password?
  </a>
  <div
    className="grass"
  />
</div>
`;
